CIP-tx-serialization-deprecation-cycles/README.md

```diff
@@ -0,0 +1,116 @@
+---
+CIP: $\pi$
```

Comment:
```suggestion
CIP: 80
```

Reply:
> insert rational joke here

355/113 = Not the famous irrational number _π_, but an incredible simulation!

Reply:
:laughing: I wish that `355133` was a four digit number so that we could use it for the CIP...

Reply:
You had 2 months to suggest adopting a 6-digit scheme for the CIP process, but I am sorry to tell you that we're about to go ahead and merge CIP-0001. 

PR welcome however 😄